internal/installation/platform_test.go

```diff
@@ -27,12 +27,12 @@ import (
 
 func Test_osArch(t *testing.T) {
 	inOS, inArch := runtime.GOOS, runtime.GOARCH
-	outOS, outArch := osArch()
-	if inOS != outOS {
-		t.Errorf("returned OS=%q; expected=%q", outOS, inOS)
+	out := OSArch()
+	if inOS != out.OS {
+		t.Errorf("returned OS=%q; expected=%q", out.OS, inOS)
```

Comment:
Use cmp.diff here
What if we add another field :)

Reply:
Updated.